tests/test_cord_processor.py

```diff
@@ -0,0 +1,109 @@
+import json
+import logging
+import mock
+
+from requests import Response
+import structlog
+from sdc.rabbit.exceptions import QuarantinableError, RetryableError
+from structlog.stdlib import LoggerFactory
+from structlog.threadlocal import wrap_dict
+import unittest
+from unittest.mock import MagicMock
+
+from app import settings
+from app.helpers.sdxftp import SDXFTP
+from app.processors.cord_processor import CordProcessor
+from tests.test_data import cord_survey
+
+ftpconn = SDXFTP("", "", "")
+
+
+class TestCoraProcessor(unittest.TestCase):
```

Comment:
I think it's probably worth inheriting from a base class here, there is a lot of shared code between this and the test_cora_processor.

Reply:
Even the class name is the same! :)